ci/dependency-update/Jenkinsfile

```diff
@@ -7,44 +7,44 @@
 
 @Library('hibernate-jenkins-pipeline-helpers@1.5') _
 
-// NOTE: Remember to update the matrix axes below when adding/removing entries here.
+def configurations = [
```

Comment:
We might want to port these changes to `main`, even if we don't have a use for them there. Just so that we're ready for similar requirements on newer branches.